queue_job/models/queue_job.py

```diff
@@ -161,11 +161,29 @@ def _change_job_state(self, state, result=None):
             job_.store()
 
     @api.multi
-    def button_done(self):
-        result = _('Manually set to done by %s') % self.env.user.name
+    def action_done(self, reason=None):
+        result = _(
+            u"Manually set to done by {}"
+        ).format(self.env.user.name)
+        if reason:
+            result = _(
+                u"{} with reason : {}"
```

Comment:
```suggestion
                u"{} with reason: {}"
```

Usually no space before colon in English